iohub/display_utils.py

```diff
@@ -0,0 +1,139 @@
+# Utility functions for displaying data
+
+from typing import Tuple
+
+import numpy as np
+
+from iohub.ngff_meta import ChannelMeta, WindowDict
+
+def rgb_to_hex(r: int, g: int, b: int) -> str:
+    """
+    Convert rgb values to hex
+    Example: Red (255,0,0) -> 'FF0000'
+    Parameters
+    ----------
+    r : int
+        red
+    g : int
+        green
+    b : int
+        blue
+    Returns
+    -------
+    str: Hex string corresponding to the RGB value
+    """
+    hex_color = "{:02X}{:02X}{:02X}".format(r, g, b)
+    return hex_color
+
+# Dictionary with key works and most popular fluorescent probes
+channel_colors = {
+    "lime": ["GFP", "Green", "Alexa488", "GCaMP","FITC", "mNeon"],
```

Comment:
green and magenta complement each other, i.e., their overlay is white. Why choose slightly lighter lime?

Reply:
This is also a good point. Initially, I had a typo in my green set as (0,128,0) which made it darker than lime green. Fixed now. Thank you!